api/admin/opds.py

```diff
@@ -135,9 +136,13 @@ def suppressed(cls, _db, title, url, annotator, pagination=None):
         pagination = pagination or Pagination.default()
 
         q = _db.query(LicensePool).filter(
-            LicensePool.suppressed == True).order_by(
-                LicensePool.id
+            and_(
+                LicensePool.suppressed == True,
+                LicensePool.superceded == False,
```

Comment:
This changes the hidden books in the admin interface. Before a book would show up when any of its license pools was suppressed. If the non-superceded license pool wasn't suppressed, it appeared in the main catalog as well as hidden books and it wasn't possible to remove it from the list of hidden books.